packages/web-components/src/components/ic-tooltip/ic-tooltip.stories.mdx

```diff
@@ -31,6 +31,43 @@ import readme from "./readme.md";
   </Story>
 </Canvas>
 
+### Silent
+
+<Canvas>
+  <Story name="Silent" height="100px">
+    {(args) =>
+      html`<ic-tooltip
```

Comment:
Hovering over this button produces two tooltips. One generated from the browser and the other, an `ic-tooltip`